src/catalog.cpp

```diff
@@ -602,8 +602,10 @@ void Catalog::SetFileName(const wxString& fn)
 
 bool Catalog::IsFromCrowdin() const
 {
-    if((m_crowdinFileId > 0 && m_crowdinProjectId) > 0
```

Comment:
Yep, that was definitely my typo during refactoring and I would never make above intentently :), it intended to be `(m_crowdinFileId > 0 && m_crowdinProjectId > 0)`